src/main/scala/lc/core/config.scala

```diff
@@ -71,7 +84,7 @@ class Config(configFilePath: String) {
         (AttributesEnum.CAPTCHA_EXPIRY_TIME_LIMIT.toString -> 5) ~
         (AttributesEnum.THROTTLE.toString -> 1000) ~
         (AttributesEnum.THREAD_DELAY.toString -> 2) ~
-        (AttributesEnum.PLAYGROUND_ENABLED.toString -> true) ~
+        (AttributesEnum.PLAYGROUND_ENABLED.toString -> "true") ~
```

Comment:
Hmm.. changing the type will break existing configurations.

How about using the `extract` function using a case class. Something like:

```
case class ConfigFields(
  port: Option[Int],
  address: Option[String],
  playgroundEnabled: Option[Boolean],
  ...
)

val fields = configJson.extract[ConfigFields]

val port = fields.port.getOrElse(8888)
val address = fields.address.getOrElse("0.0.0.0")
val playgroundEnabled = fields.playgroundEnabed.getOrElse(true)
```

Reply:
Should this be string?